pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/test/java/org/apache/pinot/plugin/stream/kafka20/KafkaStreamLevelStreamConfigTest.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.plugin.stream.kafka20;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.pinot.spi.stream.StreamConfig;
+import org.apache.pinot.spi.stream.StreamConfigProperties;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class KafkaStreamLevelStreamConfigTest {
+  private static final String KAFKA_DECODER_CLASS_NAME =
+      "org.apache.pinot.plugin.inputformat.avro.KafkaAvroMessageDecoder";
+
+  private StreamConfig getStreamConfig(String topic, String bootstrapHosts, String buffer, String socketTimeout) {
+    return getStreamConfig(topic, bootstrapHosts, buffer, socketTimeout, null, null, null);
+  }
+
+  private StreamConfig getStreamConfig(String topic, String bootstrapHosts, String buffer, String socketTimeout,
+      String fetcherSize, String fetcherMinBytes, String isolationLevel) {
+    return getStreamConfig(topic, bootstrapHosts, buffer, socketTimeout, fetcherSize, fetcherMinBytes, isolationLevel,
+        null);
+  }
+
+  public StreamConfig getStreamConfig(String topic, String bootstrapHosts, String buffer, String socketTimeout,
```

Comment:
Totally agree with you. Can you help me understanding why we are moving the in `pinot-kafka-base` instead of `utils` [here](https://github.com/tanmesh/pinot/tree/db07f2928a10907badd6b4c1ed583020e1b2d9d2/pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/test/java/org/apache/pinot/plugin/stream/kafka20/utils)?

Reply:
@tanmesh My suggestion was to move it to a common utils class so that the code can be re-used. You can move it to `utils`. That works too.